tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -210,27 +210,6 @@ trait WskWebActionsTests
             authorizedResponse.body.asString shouldBe namespace
     }
 
-    if (testRoutePath == "/api/v1/web") {
```

Comment:
Added the unit test back without the conditional statement that prevented it from running.